compiler/test/dotty/tools/repl/ScriptedTests.scala

```diff
@@ -60,6 +60,16 @@ class ScriptedTests extends ReplTest with MessageRendering {
         case nonEmptyLine => nonEmptyLine :: Nil
       }
 
+    def printOutput(prefix: String, output: String): Unit = {
+      val bytes = output.getBytes
+      println(prefix+
+        " (#chars="+output.length+
+        ", #CR="+(bytes.count(_ == 13))+
+        ", #LF="+(bytes.count(_ == 10))+
+        ") ========>"+EOL+
+        output.replaceAll("\r", "<CR>").replaceAll("\n", "<LF>"+EOL))
```

Comment:
I would rather not have these replacements done in the printed output, it makes it harder to fix a broken test by copy-pasting the console output in a file.